client/app/scripts/liveblog-edit/module.js

```diff
@@ -20,18 +20,16 @@ define([
     BlogEditController.$inject = [
         'api', '$q', '$scope', 'blog', 'notify', 'gettext',
         'upload', 'config', '$rootScope', 'embedService', 'postsService', 'modal',
-        'blogService'
+        'blogService', '$interval'
```

Comment:
$interval should be injected in the https://github.com/superdesk/liveblog/commit/970890fcd444639315fd4c15593bb64e6e8484f4